app/src/main/java/com/internship/tvseries/data/repository/topRatedRepository/TopRatedRepositoryLynxTv.java

```diff
@@ -0,0 +1,11 @@
+package com.internship.tvseries.data.repository.topRatedRepository;
+
+import com.internship.tvseries.data.api.MovieApi;
+
+public class TopRatedRepositoryLynxTv extends BaseClassTopRatedRepository {
```

Comment:
Base class Popular and Top Rated are duplicated.
The idea of the base class is to keep the same logic into one class only, so the TopRatedRepositoryLynxTv class should extend the BaseClassPopularRepository.
Please don't use the "Class" in the name of a class. Change the BaseClassPopularRepository to  BaseClassPopularRepository and if you want to make these changes, rename the base class to be BaseTvRepository